tests/robustness/failpoint/cluster.go

```diff
@@ -304,3 +308,28 @@ func patchArgs(args []string, flag, newValue string) error {
 	}
 	return fmt.Errorf("--%s flag not found", flag)
 }
+
+type growMember struct {
+	currentSize int
+	targetSize  int
+}
+
+func (f *growMember) Inject(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.EtcdProcessCluster, baseTime time.Time, ids identity.Provider) ([]report.ClientReport, error) {
+	for len(clus.Procs) < f.targetSize {
+		_, err := clus.StartNewProc(ctx, nil, t, false)
+		if err != nil {
+			return nil, err
+		}
+		f.currentSize++
+		time.Sleep(etcdserver.HealthInterval)
+	}
+	return nil, nil
+}
+
+func (f *growMember) Name() string {
+	return "GrowCluster"
+}
+
+func (f *growMember) Available(config e2e.EtcdProcessClusterConfig, member e2e.EtcdProcess, profile traffic.Profile) bool {
+	return f.currentSize < f.targetSize
```

Comment:
Here is the tricky part. If you start from cluster of size 1, then the traffic will talk to the 1 initial member. I would recommend instead to start to from 3 members, remove 2 and add back 2. This is still tricky because `StartNewProc` will allocate unique port numbers so we will need to work around that.

Reply:
Yep.  How about the current one?